compiler/src/iree/compiler/Codegen/LLVMGPU/test/vector_distribution_pipeline_test.mlir

```diff
@@ -0,0 +1,31 @@
+// RUN: iree-opt --split-input-file \ 
+// RUN: --iree-codegen-llvmgpu-use-vector-distribution '--pass-pipeline=builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-select-lowering-strategy, iree-llvmgpu-lower-executable-target, canonicalize)))' %s | FileCheck %s
```

Comment:
The check-resource-usage pass is actually run during the lowering to llvm, which this pass pipeline doesn't run. But, I'm not sure what else to use here...

This test does confirm the e2e working of the fit shared memory thing though. Also, this test would not pass the fixed check resource usage pass without the fit shared memory changes.